wildfly-manager/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM jboss/wildfly:20.0.1.Final
+
+MAINTAINER florian.volk@scheer-group.com
+EXPOSE 8080 8443
+
+ARG APIMAN_VERSION=2.1.0.Final
```

Comment:
Compared to the other Dockerfiles this is the only one where a APIMAN_VERSION is set

Reply:
Thanks, I missed that